dev/source/docs/using-sitl-for-ardupilot-testing.rst

```diff
@@ -60,12 +60,12 @@ location in the
 `ardupilot/Tools/autotest/locations.txt <https://github.com/ArduPilot/ardupilot/blob/master/Tools/autotest/locations.txt>`__
 file.
 
-For example, to start Copter in *Ballarat* (a named location in
+For example, to start the simulator in *Ballarat* (a named location in
 **locations.txt**) call:
 
 ::
 
-    cd ArduCopter 
+    cd ardupilot/Tools/autotest/

```

Comment:
This is a fair indication that sim_vehicle.py is not in your path.

Being in the same directory as sim_vehicle.py *should not change that*; `.` should never be in your path.

I think this would be better formulated without the `cd`, at all:
```
./Tools/autotest/sim_vehicle.py -v ArduCopter ...
```